pkgs/development/libraries/science/chemistry/cppe/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, cmake
```

Comment:
Instead of making this file generate both the Python bindings and the regular C bindings, maybe you'd consider following the approach that I used in my NUR with the similar (in terms of packaging) ``tweedledum`` library:
* 1 Nix module for the C library: https://github.com/drewrisinger/nur-packages/blob/8c70c747b48bd8b13eb3173aa27d6afdb7f694b1/pkgs/libraries/tweedledum/default.nix
* 1 Nix module for the python library, which inherits the source from the C library: https://github.com/drewrisinger/nur-packages/blob/8c70c747b48bd8b13eb3173aa27d6afdb7f694b1/pkgs/python-modules/tweedledum/default.nix

It's slightly more Nix packaging, but I doubt you'd run into any python version conflicts this way.

Reply:
This was a very good solution. Using the setup.py build solved all the python problems.